### ISSUE TITLE
"from now" style times are not supported

### ISSUE THREAD

**Issue by devlace** [wanasit/chrono#123]
It would be great if Chronos can understand the following phrases:

"4 hours from now"
"2 days from now"
"A month from now"

Currently, it seems to only pick up "now" and returns the current datetime.

**Comment by penne12** [comment]
We could do something like the [time ago format parser][tafp]: 

[tafp]: https://github.com/wanasit/chrono/blob/master/src/parsers/EN/ENTimeAgoFormatParser.js

**Comment by wanasit** [comment]
@penne12 agree. it look like time ago parser. I'm not sure if creating a new parser or updating current parsers is a better solution though.

**Comment by wanasit** [comment]
Sorry for the slow reply. This has already been added in the more recent versions.
